credentials-migration/README.md

```diff
@@ -0,0 +1,27 @@
+# Jenkins Credentials Migration
+
+The following scripts are created to migrate the credentials at Root (System) and Folder level from one Jenkins Master to another one.
+
+In Jenkins, `$JENKINS_HOME/secrets/master.key` is the secret used to encrypt all the credentials stored on disk. The new instances created will have their own `$JENKINS_HOME/secrets/master.key`, so it is necessary to export/import the credentials to re-encrypt them in the new Jenkins master.
```

Comment:
I think that it could be a good idea to put a special note on this procedure as the credentials will be exposed so that system admins can pay special attention to it.